docs/team/florentianayuwono.md

```diff
@@ -3,44 +3,48 @@ layout: page
 title: Florentiana Yuwono's Project Portfolio Page
 ---
 
-### Project: SoConnect
+# Project: SoConnect
 
-SoConnect is a desktop app for NUS SoC students to manage their contacts, optimised for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+SoConnect is a **desktop app for managing contacts and tasks**. It aims to help NUS SoC students stay better connected to their school life, in terms of social connections and school tasks. SoConnect is also **optimized for use via a Command Line Interface (CLI)** while still having the benefits of a Graphical User Interface (GUI). The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 42 kLoC.
 
-Given below are my contributions to the project.
+## Contribution
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+### New Feature: Advanced Search Functionality
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
+**What it does**: Allows user to have a mini Google inside SoConnect. The search command can generate relevant search result, even in the event of mistyping or only partial information is available.
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=florentianayuwono&breakdown=true)
+**Justification**: This feature improves the user productivity significantly because a user will not need to provide full information upon searching and the app could still return accurate and relevant result based on the query. 
 
-* **Project management**:
-  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+**Highlight**: This enhancement replaces the previous `find` functionality. The search feature itself requires careful design on how the algorithm works, especially on the relevancy reduction level.
```

Comment:
Perhaps use `feature` rather than `functionality`? To standardise the phrasing.

Reply:
Yes correct that sounds better